src/Compilers/Test/Core/Assert/AssertEx.cs

```diff
@@ -838,11 +838,19 @@ private sealed class LineComparer : IEqualityComparer<string>
             public int GetHashCode(string str) => str.Trim().GetHashCode();
         }
 
-        public static void AssertLinesEqual(string expected, string actual, string message, string expectedValueSourcePath, int expectedValueSourceLine, bool escapeQuotes)
-        {
-            IEnumerable<string> GetLines(string str) =>
+        private static IEnumerable<string> GetLines(string str) =>
                 str.Split(new[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries);
 
+        public static void AssertLinesEqual(string expected, string actual)
```

Comment:
For context. Something about my testing workflow was causing trailing whitespace on lines of the test to be lost, but our asserts were sensitive to that whitespace. I was finding it really frustrating to dig up that whitespace (basically had to go into the debugger and poke at the actual documentation string) so went ahead and started using a whitespace-insensitive comparison (though still line-sensitive) for a few of the tests.